models/eos/interfaces/deleted_example_01.txt

```diff
@@ -0,0 +1,34 @@
+
+# Using deleted
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Loopback1
+# !
+# interface Management1
+#    ip address dhcp
+# !
+
+- name: Configure interfaces
```

Comment:
Should we provide a better explanation for the module invocation?  Maybe something like "Delete or return to defaults the description, speed, duplex MTU etc of all interfaces" ?